macos/package_files/postinstall.sh

```diff
@@ -119,17 +119,17 @@ rm -rf ${DIR}/packages_files
 # Remove old ossec user and group if exists and change ownwership of files
 
 if [[ $(dscl . -read /Groups/ossec) ]]; then
-  find ${DIR}/ -group ossec -user root -exec chown root:wazuh {} \ > /dev/null 2>&1 || true
+  find ${DIR}/ -group ossec -user root -print0 | xargs -0 chown root:wazuh > /dev/null 2>&1 || true
```

Comment:
In my macOS env the `xargs` command is not available.
```
bash-3.2# find /Library/Ossec/agentless -group wazuh -user root -print0 | xargs -0 chown ossec:ossec
bash:  xargs: command not found
bash-3.2# ls -lah /Library/Ossec/agentless/
total 120
drwxr-x---  14 root  wazuh   448B Dec  5 17:28 .
drwxr-x---  15 root  wazuh   480B Dec  5 17:28 ..
-rwxr-x---   1 root  wazuh   2.4K Nov 29 10:43 main.exp
-rwxr-x---   1 root  wazuh   2.3K Nov 29 10:43 register_host.sh
-rwxr-x---   1 root  wazuh   1.4K Nov 29 10:43 ssh.exp
-rwxr-x---   1 root  wazuh   5.2K Nov 29 10:43 ssh_asa-fwsmconfig_diff
-rwxr-x---   1 root  wazuh   5.3K Nov 29 10:43 ssh_foundry_diff
-rwxr-x---   1 root  wazuh   898B Nov 29 10:43 ssh_generic_diff
-rwxr-x---   1 root  wazuh   1.1K Nov 29 10:43 ssh_integrity_check_bsd
-rwxr-x---   1 root  wazuh   1.1K Nov 29 10:43 ssh_integrity_check_linux
-rwxr-x---   1 root  wazuh   1.6K Nov 29 10:43 ssh_nopass.exp
-rwxr-x---   1 root  wazuh   5.1K Nov 29 10:43 ssh_pixconfig_diff
-rwxr-x---   1 root  wazuh   915B Nov 29 10:43 sshlogin.exp
-rwxr-x---   1 root  wazuh   1.3K Nov 29 10:43 su.exp
bash-3.2#
```

Reply:
Finally the fix will be applied for the manager only. Agents don't have this problem.